lib/rucio/tests/test_multi_vo.py

```diff
@@ -663,12 +668,35 @@ def test_recover_vo_denied(self):
         response = self.rest_client.post('/vos/' + self.vo['vo'] + '/recover', headers=headers(auth(token)), json=params)
         assert response.status_code == 401
 
+    def test_rest_vomap(self):
```

Comment:
This needs a noparallel mark, since the accounts in the vo can change from other tests in-between checking them. Happened on my end: https://github.com/bziemons/rucio/runs/2894657636?check_suite_focus=true#step:4:5341

Reply:
Sure, I've added the noparallel mark to this test. Thanks for sorting out the conflicts and headers.